org.eventb.emf.persistence/src/org/eventb/emf/persistence/SaveResourcesCommand.java

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
```

Comment:
This SaveResourcesCommand does not depend on Event-B EMF at all. Would it be better to have it in a separate feature/plugin?

Reply:
True, but we don't have a feature/plugin for generic EMF utilities. Lets leave it here for the time being. Added a TODO in 9470997